### PR TITLE
URL path and renaming page title's 

### PR DESCRIPTION
**Pull request by nataliasabadysh**
### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BWT-131
https://bcparksdigital.atlassian.net/browse/CM-201

### CODE REVIEW

File: src/staging/src/pages/active-advisories.js [hunk 1]
```diff
@@ -338,7 +338,7 @@ const PublicAdvisoryPage = ({ data }) => {
   return (
     <>
       <Helmet>
-        <title>BC Parks | Public Advisories</title>
+        <title>BC Parks | Active advisories</title>
```
Comment (foley-lynn) [inline src/staging/src/pages/active-advisories.js:341]: You didn't know this, but this should be swapped so it's:

Active advisories | BC Parks

**Review by foley-lynn — CHANGES_REQUESTED** [review]
One small change, and you've also got conflicts to resolve.

File: src/staging/src/pages/active-advisories.js [hunk 2]
```diff
@@ -338,7 +338,7 @@ const PublicAdvisoryPage = ({ data }) => {
   return (
     <>
       <Helmet>
-        <title>BC Parks | Public Advisories</title>
+        <title> Active advisories | BC Parks </title>
```
Comment (ayumi-oxd) [inline src/staging/src/pages/active-advisories.js:341]: Sorry for nitpicking, I'm just wondering if you could remove spaces between title tags. It wouldn't cause any issues but would look nicer IMHO.

File: src/staging/src/pages/sitemap.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const SitemapPage = ({ data }) => {
   return (
     <>
       <Helmet>
-        <title>Site Map | BC Parks</title>
+        <title>BC Parks | Site Map </title>
```
Comment (ayumi-oxd) [inline src/staging/src/pages/sitemap.js:28]: The same as above, space before </title>
Also, is this OK to have "BC parks" first?

File: src/staging/src/templates/park.js [hunk 4]
```diff
@@ -253,7 +253,7 @@ export default function ParkTemplate({ data }) {
   return (
     <div className="grey-background">
       <Helmet>
-          <title>BC Parks | {park.protectedAreaName}</title>
+          <title>{park.protectedAreaName} | BC Parks </title>
```
Comment (ayumi-oxd) [inline src/staging/src/templates/park.js:256]: The same as above, space before </title>

File: src/staging/src/pages/active-advisories.js (hunk 2 above)
Comment (foley-lynn) [reply src/staging/src/pages/active-advisories.js:341]: Actually, it's a good point. It might affect the page title in browser search results. Good catch!

**Review by foley-lynn — APPROVED** [review]
Approved once you change those issues that Ayumi pointed out and fix the merge conflicts.

**Pull request merged**